tbx/core/blocks.py

```diff
@@ -1050,11 +1050,21 @@ class Meta:
         label_format = "{headline_number} {description} {further_details}"
 
 
-class NumericStatisticsGroupBlock(blocks.StructBlock):
+class AbstractStatisticsGroupBlock(blocks.StructBlock):
```

Comment:
I feel like this could be shared by more blocks since an optional title and intro is pretty common, but for now this is fine. :)